CONFIGURATION.md

```diff
@@ -0,0 +1,128 @@
+# Configuring NTPD-rs
+
+NTPD-rs is an implementation of NTP completely written in Rust, with a focus on exposing a minimal attack surface. The project is currently in an early stage, but if you want to try out NTPD-rs on a non-critical system, this guide provides the basic information needed to properly build and configure NTPD-rs.
+
+## Limitations
+
+The current implementation has several important limitations which it is important to be aware of. First of all, the current implementation is an NTP client only, and does not support acting as an NTP server.
+
+Second, there are currently a few limitations around configuring connections to other NTP nodes:
+ - Only Client-Server connections are supported
+ - DNS lookup is currently only done at startup. Changes in the IP address 
+ - There is no support for NTP pools yet. Multiple 
+ - Changes in network interfaces are not picked up dynamically and will require a reboot of the daemon.
+
+Finally, please be aware that this is very early stage software, and is not yet ready for production software. This software SHOULD NOT be used on any system where you cannot handling experiencing issues, either reliability or security related.
```

Comment:
I think the first sentence here is too close to the first section to be useful. The second sentence might be good to keep, but is kind of buried here in the end of the section where it's less likely to be actually read.

"cannot handling experiencing" issues.

Reply:
fixed, moved warning up to first paragraph